src/Linters/DisableASTLinter.php

```diff
@@ -0,0 +1,129 @@
+<?hh // strict
+/**
+ * Copyright (c) 2016, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional
+ * grant of patent rights can be found in the PATENTS file in the same
+ * directory.
+ *
+ */
+
+namespace Facebook\HHAST\Linters;
+
+use type Facebook\HHAST\EditableNode;
+use type Facebook\HHAST\EditableToken;
+use type Facebook\HHAST\IControlFlowStatement;
+use type Facebook\HHAST\BreakStatement;
+use type Facebook\HHAST\ContinueStatement;
+use type Facebook\HHAST\EchoStatement;
+use type Facebook\HHAST\GotoStatement;
+use type Facebook\HHAST\ReturnStatement;
+use type Facebook\HHAST\ThrowStatement;
+use type Facebook\HHAST\TryStatement;
+use type Facebook\HHAST\UnsetStatement;
+use namespace Facebook\HHAST;
+use namespace HH\Lib\Str;
+use namespace HH\Lib\Vec;
+use namespace HH\Lib\C;
+
+class DisableASTLinter {
```

Comment:
This looks like it could/should be a collection of functions in a namespace - no properties, I don't see a need for this to be instantiable

Reply:
Also, maybe rename to indicate that it suppresses lint errors, rather than implying that it can only disable the entire linter.